circuitbreakerpanel/circuitbreakerpanel-java-mvn/README.md

```diff
@@ -0,0 +1,38 @@
+#CircuitBreakerPanel recipe for lagom with java
```

Comment:
This doesn't format correctly on GitHub (you can click the "View" link to see).

GitHub requires whitespace between the `#` and the heading text.

Also, please capitalize "Lagom" and "Java".

Reply:
oh, didn't realise that. Thanks!